sql/core/src/main/scala/org/apache/spark/sql/KeyValueGroupedDataset.scala

```diff
@@ -112,6 +115,30 @@ class KeyValueGroupedDataset[K, V] private[sql](
         logicalPlan))
   }
 
+  def flatMapRGroups(
+    func: Array[Byte],
```

Comment:
Keep consistent with the naming rules in the other function's parms
`func` -> `f`